.github/workflows/deploy_app.yml

```diff
@@ -0,0 +1,43 @@
+name: Deploy to Staging
```

Comment:
I am a bit confused by the name, is this a staging deploy or a production deploy?

Reply:
It's both, cause there's only one! Oops, changing to production for consistency.